tests/readme-icons.test.js

```diff
@@ -0,0 +1,34 @@
+import { test, exec } from 'uvu';
+import * as assert from 'uvu/assert';
+import path from 'path';
+import * as fs from 'fs';
```

Comment:
Use the `node:` protocol:

```suggestion
import path from 'node:path';
import { promises as fs } from 'node:fs';
```

Reply:
Don't know what difference does this makes, but sure.